### DISCUSSION TITLE
Barcode Scanner

### DISCUSSION THREAD

**Discussion by pavankalyan-rachagundla** [dotnet/maui#6586] [General]
Hi team,
  Can anyone help me , How to create a barcode scanner in .Net maui by using Camera.
Iam already tried with ZXing library, Aloha barcode library so many.. but no use.

Could you provide any suggestions how to implement barcode scanning in .Net MAUI.

Thanks in advance❤

Thanks,
R PAVANKALYAN.

**Comment by xtuzy** [comment]
Try this? https://github.com/Redth/BigIslandBarcoding/issues/14 . if you maui version is new, you need try download it and rebuild by youself, maui have something change between different version.

**Comment by nextcodelab** [comment]
There is an issue in ZXing.. the autofocus is sometimes not working and make the library not usable...
